.github/workflows/screener-run.yml

```
@@ -1,32 +1,49 @@
 name: Screener run
-
 on:
-  workflow_run:
-    workflows:
-      - Screener build
-    types:
-      - completed
+  pull_request:
 
+  #workflow_run:
+  #  workflows:
+  #    - Screener build
+  #  types:
+  #    - completed
 env:
   AZURE_STORAGE_CONNECTION_STRING: ${{secrets.AZURE_STORAGE_CONNECTION_STRING}}
-
 jobs:
   screener-react-northstar:
     runs-on: 'ubuntu-latest'
-
     name: Screener @fluentui/react-northstar
-
     steps:
       - uses: actions/checkout@v3
         with:
-          ref: ${{github.event.workflow_run.head_branch}}
           fetch-depth: 0
 
       - uses: actions/setup-node@v3
         with:
           node-version: 14.18.1
           cache: 'yarn'
 
+      - name: Check if test app artifact deployed
+        uses: actions/github-script@v6
+        id: skip-screener
+        with:
+          script: |
+            let allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: 2913281075,
+            });
+            let matchArtifact = allArtifacts.data.artifacts.filter((artifact) => {
+              return artifact.name == "northstar-artifact"
+            })[0];
+            if(matchArtifact !== undefined)
+              return 'false'
+            return 'true'
+          result-encoding: string
+
+      - name: Set env variable for skipping screener
+        run: echo "IS_ARTIFACT_PRESENT=${{steps.skip-screener.outputs.result}}" >> $GITHUB_ENV
```

```suggestion
            github.core.exportVariable('IS_ARTIFACT_PRESENT', (matchArtifact !== undefined).toString());
```

Please check, but it seems that [`.exportVariable()`](https://github.com/actions/toolkit/tree/main/packages/core#exporting-variables) could work there better.

---

I will try this